src/Background/estimators.jl

```diff
@@ -133,3 +133,61 @@ end
 
 MMM() = MMM(3, 2)
 estimate_background(alg::MMM, data; dims = :) = alg.median_factor * median(data, dims = dims) - alg.mean_factor * mean(data, dims = dims)
+
+"""
+    Biweight_location(M=nothing, c=6.0)
+
+Estimate the background using robust statistic biweight location.
+
+# Example
+```jldoctest
+julia> x = ones(3,5);
+
+julia> estimate_background(Biweight_location, x)
+1.0
+
+julia> estimate_background(Biweight_location, x, dims = 1)
+1×5 Array{Float64,2}:
+ 1.0  1.0  1.0  1.0  1.0
+ ```
+"""
+struct Biweight_location{T <: Number} <: BackgroundEstimator
+    M::Union{Nothing, T}
+    c::T
+end
+
+Biweight_location() = Biweight_location(nothing, 6.0)
+
+function Biweight_util(data::Array{T}, M::Union{Nothing, T}, c::T) where T <: Number
```

Comment:
function names should be lowercase. Also this is calculating the biweight location, so perhaps we name it as such instead of util?
```julia
biweight_location
```
also, we don't need to type parametrize this, as well we should accept abstract types
```julia
function biweight_location(data::AbstractArray, c, M=median(data))
```

Reply:
Since this function won't be user-accessible, is it necessary to assign `M = median(data)`? also we are already checking that inside the function, and if `M === nothing` then we assign M to be `median(data)`.